sdk/src/identity/identity_assertion/assertion.rs

```diff
@@ -333,25 +333,38 @@ impl IdentityAssertion {
             // Load the trust handler settings. Don't worry about status as these
             // are checked during setting generation.
 
-            if let Ok(Some(ta)) = get_settings_value::<Option<String>>("cawg_trust.trust_anchors") {
-                let _ = ctp.add_trust_anchors(ta.as_bytes());
-            }
+            let cose_verifier =
+                if let Ok(true) = get_settings_value::<bool>("cawg_trust.bypass_trust") {
```

Comment:
why not just not configure cawg trust if you don't want it?
i.e. you can update the settings to not enable cawg trust lists.
If we need a special setting for this, I'd rather have cawg_trust enabled  by default and then disable it. it just seems clearer to me.